app/services/ExactTargetService.scala

```diff
@@ -5,7 +5,6 @@ import com.amazonaws.services.sqs.AmazonSQSClient
 import com.amazonaws.services.sqs.model._
 import com.github.nscala_time.time.Imports._
 import com.gu.aws.CredentialsProvider
-import com.gu.lib.Retry
```

Comment:
this is the only substantial change to the imports, the rest was just intellij's idea of optimal import order